MetaMask/metamask-mobile
# fix: replace decomissioned cloudflare-ipfs.com with gateway.pinata.cloud

## **Description**

The default IPFS gateway cloudflare-ipfs.com was decomissioned on August 14 2024.

https://blog.cloudflare.com/cloudflares-public-ipfs-gateways-and-supporting-interplanetary-shipyard/

- Change default IPFS gateway from `cloudflare-ipfs.com` to `gateway.pinata.cloud`
- Remove `cloudflare-ipfs.com` from IPFS gateways

## **Related issues**


## **Manual testing steps**
1. Set up wallet on previous version, keeping default IPFS gateway
2. Observe IPFS gateway requests failing
2. Upgrade metamask-mobile with this patch
3. Observe IPFS gateway requests succeeding

## **Screenshots/Recordings**


### **Before**

### **After**

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
